cli/flags.rs

```diff
@@ -1016,6 +1017,20 @@ fn test_subcommand<'a, 'b>() -> App<'a, 'b> {
         .takes_value(true)
         .help("Run tests with this string or pattern in the test name"),
     )
+    .arg(
+      Arg::with_name("shuffle")
+        .long("shuffle")
+        .value_name("NUMBER")
+        .help("(UNSTABLE): Shuffle the order in which the tests are run")
+        .min_values(0)
+        .max_values(1)
+        .require_equals(true)
```

Comment:
It's strange that `--jobs <value>` does not require an equals, but this does. Should we require equals here? It seems like jobs is the odd one out though.

Reply:
Aligns with the `--inspect` flag, maybe we should go back and be stricter with --jobs but jobs has no required equals for `make` compatibility (for example `make -j3`).

For reference, inspect has the same setup with min 0, max 1, requires value:
```rs
.arg(
  Arg::with_name("inspect")
    .long("inspect")
    .value_name("HOST:PORT")
    .help("Activate inspector on host:port (default: 127.0.0.1:9229)")
    .min_values(0)
    .max_values(1)
    .require_equals(true)
    .takes_value(true)
 ```